tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Example test. Update/remove/ignore as needed."""
```

Comment:
The `tests/` directory used by Pytest isn't a Python module by default. Making it one is a useful hack for sharing code between tests/fixtures, but probably not something we should promote as a default. We can remove `tests/__init__.py`.